metadata_service/api/user.py

```diff
@@ -3,7 +3,7 @@
 
 from flask_restful import Resource, fields, marshal
 
-from metadata_service.api.table import table_detail_fields
+from metadata_service.api.popular_tables import popular_table_fields
```

Comment:
Can you rename `popular_table_fields` to something generic like `table_summary_fields` and move it to a common file for tables in general?

Reply:
Also I think `popular_table_fields` doesn't include `last_updated_epoch`. I think the data model used by search service is more complete.

Reply:
this has been used by atlas proxy as well which we could do but will be in a different pr.